app/src/components/UserProfileSettingsDrawer.js

```diff
@@ -157,6 +157,22 @@ class UserProfileSettingsDrawer extends React.Component {
 		}
 	}
 
+	downloadOPML() {
+		fetch('GET', `/opml/download`)
+			.then(res => {
+				if (res.data) {
+					const link = document.createElement('a');
+					const blob = new Blob([res.data], { type: 'text/xml' });
+					link.href = URL.createObjectURL(blob);
+					link.download = 'export.xml';
+					link.click();
+				}
+			})
+			.catch(err => {
+				if (window.console) console.log(err); // eslint-disable-line no-console
```

Comment:
It would be better to at least show the user a generic message that export failed instead of failing silently